crypto/s2n_tls13_keys.h

```diff
@@ -74,12 +74,11 @@ extern const struct s2n_blob s2n_tls13_label_traffic_secret_iv;
 int s2n_tls13_keys_init(struct s2n_tls13_keys *handshake, s2n_hmac_algorithm alg);
 int s2n_tls13_keys_free(struct s2n_tls13_keys *keys);
 int s2n_tls13_derive_binder_key(struct s2n_tls13_keys *keys, struct s2n_psk *psk);
-int s2n_tls13_derive_early_secrets(struct s2n_tls13_keys *handshake, struct s2n_psk *psk);
-int s2n_tls13_derive_handshake_secrets(struct s2n_tls13_keys *handshake,
-                                        const struct s2n_blob *ecdhe,
-                                        struct s2n_hash_state *client_server_hello_hash,
-                                        struct s2n_blob *client_secret,
-                                        struct s2n_blob *server_secret);
+int s2n_tls13_derive_early_secret(struct s2n_tls13_keys *handshake, struct s2n_psk *psk);
+int s2n_tls13_derive_early_traffic_secret(struct s2n_tls13_keys *keys, struct s2n_hash_state *client_hello_hash, struct s2n_blob *secret);
```

Comment:
These similar names confuse me. Not enough to block, but I think that it should be improved.